### PR TITLE
Add baremetal virtualmedia workflow

### PR DESCRIPTION
**Pull request by andfasano**
Added a new workflow to test `e2e-metal-ip-virtualmedia` and updated all the existing jobs (except the one for the release-controller integration)

### CODE REVIEW

File: ci-operator/step-registry/baremetalds/e2e/virtualmedia/baremetalds-e2e-virtualmedia-workflow.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    - ref: ipi-install-rbac
+    - ref: baremetalds-devscripts-setup
+    test:
+      - ref: baremetalds-e2e-conf
+      - ref: baremetalds-e2e-test
+    post:
+    - ref: baremetalds-devscripts-gather
+    - ref: baremetalds-packet-teardown
+  documentation: |-
+    This workflow executes the common end-to-end test suite on a cluster provisioned by running dev-scripts on a packet server.
```
Comment (stbenjam) [inline ci-operator/step-registry/baremetalds/e2e/virtualmedia/baremetalds-e2e-virtualmedia-workflow.yaml:19]: Can we update this to describe that it's a virtualmedia based deployment?

File: ci-operator/step-registry/baremetalds/e2e/virtualmedia/baremetalds-e2e-virtualmedia-workflow.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+workflow:
+  as: baremetalds-e2e-virtualmedia
+  steps:    
+    env:
+      DEVSCRIPTS_CONFIG: |
+        PROVISIONING_NETWORK_PROFILE=Disabled
```
Comment (stbenjam) [inline ci-operator/step-registry/baremetalds/e2e/virtualmedia/baremetalds-e2e-virtualmedia-workflow.yaml:6]: What do you think about making this also add `IP_STACK=v4`? Disconnected adds quite some time to a deployment. Is there a reason to do this as IPv6?

Comment (andfasano) [reply ci-operator/step-registry/baremetalds/e2e/virtualmedia/baremetalds-e2e-virtualmedia-workflow.yaml:6]: I just copied the old conf file, but I agree it makes more sense

Comment (stbenjam) [reply ci-operator/step-registry/baremetalds/e2e/virtualmedia/baremetalds-e2e-virtualmedia-workflow.yaml:7]: Yea I didn't realize it was always IPv6. Thanks!

**Comment by stbenjam** [comment]
/approve

We'll also need an installer person.

**Comment by stbenjam** [comment]
/hold

Wait for at least one green.

**Comment by wking** [comment]
/lgtm

I guess if it breaks, you'd also have approver ability in your steps to self-fix, so I'm agnostic on whether you wait for a successful rehearsal.

**Pull request merged**